prov/verbs/src/fi_verbs.h

```diff
@@ -251,6 +251,13 @@ int fi_ibv_dgram_ns_service_cmp(void *svc1, void *svc2)
 	return (service1 < service2) ? -1 : (service1 > service2);
 }
 
+typedef void(*fi_ibv_fastlock_acquire_cb)(fastlock_t *lock);
+typedef void(*fi_ibv_fastlock_release_cb)(fastlock_t *lock);
```

Comment:
I would move these as generic defines to ofi_lock.h as well.

Reply:
Done